Add collectionview docs - first part (#762)
telerik/maui-docs: controls/collectionview/grouping/styling.md

---
title: Group Styling
page_title: .NET MAUI CollectionView Documentation - Group Styles
description: Check the Telerik for .NET MAUI CollectionView styling options for the Group.
position: 0
slug: collectionview-group-styling
tags: style, group, collectionview, maui, dotnet maui
---

# .NET MAUI CollectionView Group Styling

> Style the group header and the footer by setting the following properties:

Reviewer: ```suggestion
To style the group header and the footer, set the following properties:
```

* `GroupHeaderStyle`(`Style` with target type `CollectionViewGroupHeaderItemView`)&mdash;Specifies the style applied to the group header when grouping is applied.

> * `GroupFooterStyle`(`Style` with target type `CollectionViewGroupFooterItemView`)&mdash;Specifies the style applied to the group footer when `Footer` property is set and `IsGroupFooterVisible` is set to `true`.

Reviewer: ```suggestion
* `GroupFooterStyle`(`Style` with target type `CollectionViewGroupFooterItemView`)&mdash;Specifies the style applied to the group footer when the `Footer` property is set and `IsGroupFooterVisible` is set to `true`.
```
